modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.

```

Comment:
I don't know if this is implicitly assumed here: The environment which runs the virtctl  command has permissions on the OpenShift Cluster to access the VM.
(e.g. by executing oc login or setting KUBECONFIG like described in https://docs.openshift.com/container-platform/4.13/virt/virt-using-the-cli-tools.html#virt-installing-virtctl-client_virt-using-the-cli-tools )